src/main/resources/reference.conf

```diff
@@ -138,6 +138,17 @@ cassandra-journal {
   # but you cannot combine BlogPosts and Authors, since they have the same tag identifier.  
   tags {
   }
+  
+  # Minimum time between publishing messages to DistributedPubSub to announce events for a specific tag have
+  # been written. These announcements cause any ongoing getEventsByTag to immediately re-poll, rather than
+  # wait. In order enable this feature, make the following settings:
+  #  
+  #    - enable clustering for your actor system
+  #    - cassandra-journal.pubsub-minimum-interval = 10ms            (send real-time announcements at most every 10ms)
+  #    - cassandra-query-journal.eventual-consistency-delay = 0s     (so it immediately tries to show changes)
+  #
+  # Setting pubsub-minimum-interval to zero will disable the journal sending these announcements. 
+  pubsub-minimum-interval = 0 milliseconds
```

Comment:
I'd prefer to use "off" as the special disable value, we have such things in akka, e.g. https://github.com/akka/akka/blob/master/akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala#L43